skimage/measure/_regionprops.py

```diff
@@ -206,12 +206,13 @@ class RegionProperties:
     """
 
     def __init__(self, slice, label, label_image, intensity_image,
-                 cache_active, *, extra_properties=None):
+                 cache_active, *, extra_properties=None, multichannel=False):
 
         if intensity_image is not None:
-            if not intensity_image.shape == label_image.shape:
-                raise ValueError('Label and intensity image must have the'
-                                 ' same shape.')
+            ndim = label_image.ndim
+            if not intensity_image.shape[:ndim] == label_image.shape:
```

Comment:
the test should be more stringent here I think, for example we should raise also in the case of a `(100, 100)` label image and `(100, 100, 3, 4)` intensity image. Just check that there is at most one additional dimension in `intensity_image`.

Reply:
Done!